.github/workflows/macos.yml

```
@@ -26,7 +26,7 @@ jobs:
     steps:
 
       - name: Checkout sources & submodules
-        uses: actions/checkout@master
+        uses: actions/checkout@v2.4.0
```

```suggestion
        uses: actions/checkout@v2
```
It is recommended to use this tag as it following updates to the latest minor version